assets/app/scripts/controllers/build.js

```diff
@@ -70,6 +71,21 @@ angular.module('openshiftConsole')
               }
               $scope.build = build;
             }));
+            watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
+              if (action === "DELETED") {
+                $scope.alerts["deleted"] = {
+                  type: "warning",
+                  message: "Build configuration " + $scope.buildConfigName + " has been deleted."
+                };
+              }
+              $scope.buildConfig = buildConfig;
+              if ($scope.buildConfig.metadata.annotations && $scope.buildConfig.metadata.annotations["openshift.io/build-config.paused"] === "true") {
+                $scope.alerts["paused"] = {
+                  type: "warning",
+                  message: "Building from build configuration " + buildConfig.metadata.name + " has been paused."
+                };
+              }
```

Comment:
We should remove the warning when the annotation is removed.

```
else {
  delete $scope.alerts.paused; // no-op if no previous paused alerts
}
```

But I was thinking more a warning like this, which can be handled entirely in the view. (Sorry if I wasn't clear.)

https://github.com/openshift/origin/blob/master/assets/app/views/browse/build.html#L12

Maybe we don't show the paused warning if there's a status message to avoid two icons.